lib/moonshot/build_mechanism/github_release.rb

```diff
@@ -140,6 +143,20 @@ def hub_release_exists(semver)
       false
     end
 
+    def hub_find_remote_tag(tag_name)
+      attempts = 0
+      # Attempt to find the build. Rescue and re-attempt if the build can not
+      # be found on github yet.
+      begin
+        sleep 1 # Give some time for tag to showup.
```

Comment:
So, we know that github fails for N seconds all the time?  If so, this sleep should be at least N (above, this sleep was 2).  I'm fine with this being here but there should probably be a comment that we sleep first because we know that this always fails for the first N seconds.